docs/release-notes/0.10.8.md

```diff
@@ -13,3 +13,5 @@
 
 ```{rubric} Performance
 ```
+
+* Add `~anndata.experimental.read_elem_as_dask` function to handle i/o with sparse and dense arrays {pr}`1469` {user}`ilan-gold`
```

Comment:
@ivirshup Is this right? `0.10.8` or are we waiting until `0.11.0`?

Reply:
It's a feature, so I'd go `0.11`